alembic/versions/20240403134108_8f1265f626cf_.py

```diff
@@ -0,0 +1,25 @@
+"""Merge 1a712c98b3d9 and d4bd406cc25c
```

Comment:
Since the prior two migrations aren't yet committed, would it be possible to just downgrade twice and then generate a single migration file? That way, we could avoid having 3 migrations